src/context.rs

```diff
@@ -1374,14 +1374,17 @@ impl IndexMut<BlockOffset> for FrameBlocks {
   }
 }
 
+const PARTITION_CONTEXT_GRANULARITY: usize = 8;
+const MAX_B_SIZE: usize = MAX_TILE_WIDTH / PARTITION_CONTEXT_GRANULARITY;
```

Comment:
MAX_B_SIZE is still a bad name, normally a "B" is 4x4. How about "PARTITION_CONTEXT_MAX_WIDTH"? I know it's long but it's only used a few places.

Reply:
Sure, let me change it now.